app/views/orgs/lti_configurations/_form.html.erb

```diff
@@ -0,0 +1,28 @@
+<% method = local_assigns.fetch(:method) %>
+<% model = local_assigns.fetch(:model) %>
+<% allow_cancel = local_assigns.fetch(:allow_cancel, false) %>
+
+<%= form_with model: model, url: lti_configuration_path, method: method do |f| %>
+  <div class="form-group">
+    <dt><label>Learning Management System Provider</label></dt>
+    <dt>What platform does your institution use for its Learning Management System?</dt>
+    <dd><%= f.select :lms_type, lms_type_select_options, { prompt: "Select from the options listed" }, { class: "form-select" } %></dd>
+  </div>
+
+  <!--<div class="form-group">
```

Comment:
Is this commented out stuff supposed to be here?

Reply:
We actually had this field on the model as a rough idea for functionality it we currently don't need / may not ever need, and I've temporarily commented it out because it was just cluttering up the UI. Still deciding if we should leave it or delete it!

Reply:
I've decided to remove it from the UI for now, as we don't have any functionality which uses the attribute it sets.